src/main/webapp/app/course/course-scores/course-scores.component.ts

```diff
@@ -842,6 +905,14 @@ export class CourseScoresComponent implements OnInit, OnDestroy {
             emptyLine.setExerciseTypeScore(exerciseType, '');
         }
 
+        if ((this.gradingScale?.presentationsNumber ?? 0) > 0) {
```

Comment:
```suggestion
        if (this.gradingScale?.presentationsNumber) {
```
does the same as `0 | undefined| null` is falsy